rocketchat-core/src/main/java/com/rocketchat/core/ChatRoom.java

```diff
@@ -65,14 +66,28 @@ public void getMembers(RoomCallback.GetMembersCallback callback) {
     /**
      * Gets the room file list.
      *
+     * <p>Example of expected usage:
+     *
+     * <blockquote><pre>
+     * room.getFiles("0", Attachment.SortBy.UPLOADED_DATE, Sort.DESC, new PaginatedCallback() {
+     *     public void onSuccess(List list, int total) {
+     *         // Handle the file list and the total of files in the room (this is not the file list size).
+     *     }
+     *
+     *     public void onError(RocketChatException error) {
+     *        // Handle the error like showing a message to the user
+     *     }
+     * });
+     * </pre></blockquote>
+     *
      * @param offset The number of items to “skip” in the query, is zero based so it starts off at 0 being the first item.
      * @param sortBy The attribute name to sort.
      * @param sort The order in which the results should be returned.
      * @param callback The paginated callback.
-     * @since 0.8.0
      * @see BaseRoom
      * @see com.rocketchat.core.model.attachment.Attachment.SortBy
      * @see Sort
+     * @since 0.8.0
      */
     public void getFiles(String offset,
```

Comment:
Why the offset is a String?

Reply:
This is the way that it is designated on the API (our REST API). We could use an `int` and do a conversion on the final process of our SDK, but I think that it is unnecessary.

Reply:
@luciofm I changed the `offset` to int. Now we are using the `String.valueOf()` to work with our call.